src/instrumentsscene/view/parttreeitem.cpp

```diff
@@ -48,7 +48,7 @@ void PartTreeItem::init(const notation::Part* masterPart)
     }
 
     setId(part->id());
-    setTitle(part->partName().isEmpty() ? part->instrument()->name() : part->partName());
+    setTitle(part->instrument()->name());
```

Comment:
Still the part name can be different from the instrument name, like when a part contains more than one instrument, can't it? Like I usually ha a "Choir" part, containing the "Women" and Men" 'instruments'.

Reply:
I didn't know you can have multiple instruments in a single part. When you say part, do you mean like this with the separate score:
![image](https://user-images.githubusercontent.com/88136054/148554358-3cf9d00c-2735-409c-bd65-965e3a46a18f.png)

Reply:
Yes, exactly.

Reply:
But, this works as intended. I think there might be a confusion regarding the naming conventions. Though we say part to refer to the scores, its apparently different in the source code